SharpSource/SharpSource.Test/EnumWithoutDefaultValueTests.cs

```diff
@@ -81,6 +81,33 @@ public async Task EnumWithoutDefaultValue_RightName_RightValueDuplicated(string
 enum Test {{
     A = 0,
     {memberName} = 0,
+}}";
+
+        await VerifyCS.VerifyNoDiagnostic(original);
+    }
+
+    [TestMethod]
+    [DataRow("None")]
+    [DataRow("Unknown")]
+    public async Task EnumWithoutDefaultValue_RightName_NegativeValue(string memberName)
+    {
+        var original = $@"
+enum [|Test|] : short {{
```

Comment:
```suggestion
enum Test : short {{
```

Those are used to indicate where the diagnostic is but considering we don't have any, we can omit it

Reply:
There is a diagnostic. The value is not 0 as it should be.

The test ensures that the conversion code in the analyzer don't fail on negative values, but do indeed report as it should.